### PR TITLE
second part of my list of issues

### PR DESCRIPTION
**Pull request by ue71603**

### CODE REVIEW

**Review by Aurige — APPROVED** [review]

**Review by klement-MENTZ — APPROVED** [review]
change 
<xsd:documentation>Rating for traveller**r**</xsd:documentation>
to
<xsd:documentation>Rating for traveller</xsd:documentation>

**Comment by ue71603** [comment]
changed. please review @Aurige  @skinkie

**Review by skinkie — APPROVED** [review]
Thanks. Good job.

File: xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd [hunk 1]
```diff
@@ -256,7 +256,7 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Date of last trip.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="CommentsAbout" type="MultilingualString" minOccurs="0">
+			<xsd:element name="CommentsAbout" type="MultilingualString" minOccurs="0" maxOccurs="unbounded">
```
Comment (skinkie) [inline xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd:259]: This is a functional change. Where does it come from?

Comment (ue71603) [reply xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd:259]: It is. but I think it was wrong in the beginning. CommentsAbout as a single MultilingualString seems not modeled right. Remove and a separate functional change?

Comment (skinkie) [reply xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd:259]: Yes, remove, and for next we introduce it. I would even want to have a discussion with @Aurige regarding "Name" specifically for translations.

Comment (Aurige) [reply xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd:259]: And **AlternativeName** when it is specific for names

**Pull request merged**